docs-src/dev-guide/sdk-features.md

```diff
@@ -0,0 +1,571 @@
+
+## Auth (SSL/mTLS)
+
+| Feature                         | Go  | Java | TypeScript | Python | .Net | PHP |
+| ------------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Auth (SSL/mTLS)](#auth-ssltls) | Yes | Yes  | Yes        | Yes    | Yes  | Yes |
+
+Temporal SDKs support authenticating and authorizing client API calls to the Temporal server using SSL and mTLS.
+SSL (Secure Sockets Layer) and its successor TLS (Transport Layer Security) allow encrypted communication between the SDK client and the Temporal server over a network.
+mTLS (mutual TLS) is a variant where both client and server authenticate each other by verifying the provided certificates.
+This ensures secure identification and access control for API calls made from the SDK clients.
+Configuring SSL or mTLS authentication provides network encryption and protects against man-in-the-middle attacks when communicating with the Temporal server.
+
+## Built-in handler list Query
+
+| Feature                                                     | Go | Java | TypeScript | Python | .Net | PHP |
+| ----------------------------------------------------------- | -- | ---- | ---------- | ------ | ---- | --- |
+| [Built-in handler list query](#built-in-handler-list-query) | No | No   | No         | No     | No   | No  |
+
+The built-in handler list Query in Temporal allows retrieving the list of registered Signal and Query handlers for a Workflow.
+
+When this Query is sent to a Workflow, the response contains a JSON serialized list of the following information about each handler:
+
+- Handler type: either Signal or Query
+- Handler name
+- Workflow function that handles the Signal or Query
+
+This can be useful for debugging or introspecting which Signal or Query a Workflow currently supports.
+
+## Built-in Temporal CLI dev server runner
+
+| Feature                                                                             | Go  | Java | TypeScript | Python | .Net | PHP |
+| ----------------------------------------------------------------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Built-in Temporal CLI dev server runner](#built-in-temporal-cli-dev-server-runner) | Yes | No   | Yes        | Yes    | Yes  | No  |
+
+This allows starting the CLI dev server from code rather than manually from the terminal.
+It handles starting and stopping the server automatically around your Workflows.
+
+## Bulk replayer
+
+| Feature                         | Go  | Java | TypeScript | Python | .Net | PHP |
+| ------------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Bulk replayer](#bulk-replayer) | Yes | Yes  | Yes        | Yes    | Yes  | No  |
+
+The bulk replayer enables efficient parallel replay of many Workflow histories from a JSON file or an array of Workflow history event objects for testing and debugging Workflows in Temporal.
+
+## Cancellation
```

Comment:
Cancellation should not be shown as a general feature. Different forms of cancellation are so different from each other the only thing they have in common is that they are called "cancellation".